install_config/cluster_metrics.adoc

```diff
@@ -747,68 +747,12 @@ For more information about Heapster and how to access its APIs, please refer the
 link:https://github.com/kubernetes/heapster/[Heapster] project.
 endif::[]
 
-[[cluster-metrics-scaling-openshift-metrics-pods]]
-== Scaling {product-title} Metrics Pods
+[[metrics-scaling-metrics-pods]]
+== Scaling {product-title} Cluster Metrics Pods
 
-One set of metrics pods (Cassandra/Hawkular/Heapster) is able to monitor at
-least 10,000 pods.
-
-[CAUTION]
-====
-Pay attention to system load on nodes where {product-title} metrics pods run.
-Use that information to determine if it is necessary to scale out a number of
-{product-title} metrics pods and spread the load across multiple {product-title}
-nodes. Scaling {product-title} metrics heapster pods is not recommended.
-====
-
-[[cluster-metrics-scaling-pods-prereqs]]
-=== Prerequisites
-
-If persistent storage was used to deploy {product-title} metrics, then you must
-xref:../dev_guide/persistent_volumes.adoc#dev-guide-persistent-volumes[create a persistent volume (PV)]
-for the new Cassandra pod to use before you can scale out the number of
-{product-title} metrics Cassandra pods. However, if Cassandra was deployed with
-dynamically provisioned PVs, then this step is not necessary.
-
-[[cluster-metrics-scaling-pods-cassandra]]
-=== Scaling the Cassandra Components
-
-The Cassandra nodes use persistent storage, therefore scaling up or down is not possible with replication controllers.
-
-Scaling a Cassandra cluster requires you to modify the `openshift_metrics_cassandra_replicas` variable and
-re-running the xref:../install_config/cluster_metrics.adoc#deploying-the-metrics-components[deployment].
-By default, the Cassandra cluster is a single-node cluster.
-ifdef::openshift-origin[]
-To deploy more nodes, provision storage if `openshift_metrics_cassandra_replicas` equals `pv` and
-increase the `openshift_metrics_cassandra_replicas` value.
-endif::openshift-origin[]
-
-ifdef::openshift-enterprise[]
-To scale out the number of {product-title} metrics hawkular pods to two
-replicas, run:
-
-----
-# oc scale -n openshift-infra --replicas=2 rc hawkular-metrics
-----
-
-or update your inventory file and re-run the xref:../install_config/cluster_metrics.adoc#deploying-the-metrics-components[deployment].
-endif::openshift-enterprise[]
-
-[NOTE]
-====
-If you add a new node to a Cassandra cluster, the data stored in the cluster
-rebalances across the cluster. The same thing happens If you remove a node from
-the Cluster.
-====
-
-ifdef::openshift-enterprise[]
-[[cluster-metrics-horizontal-pod-autoscaling]]
-== Horizontal Pod Autoscaling
-
-{product-title} version 3.3 does not provide
-xref:../dev_guide/pod_autoscaling.adoc#dev-guide-pod-autoscaling[Horizontal
-Pod Autoscaling (HPA)] support for metrics pods and scaling metrics pods.
-endif::[]
+Information about scaling cluster metrics capabilities is available in the
+xref:../scaling_performance_capacity/scaling_cluster-metrics.adoc[Scaling and
```

Comment:
Oops. I meant to come back to that once I'd finalised the headings, etc. Good eye!